python/py-csscompressor/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-csscompressor
+version             0.9.5
+revision            0
+checksums           rmd160  23c00abcceae07604b52ec1f4cf0725f217f2640 \
+                    sha256  afa22badbcf3120a4f392e4d22f9fff485c044a1feda4a950ecc5eba9dd31a05 \
+                    size    237808
+
+categories-append   www
+license             BSD
+platforms           darwin
+supported_archs     noarch
+maintainers         {isi.edu:calvin @cardi} openmaintainer
+
+description         Python implementation of the YUI CSS Compressor
+long_description    Python implementation and almost exact port of \
+                    Yahoo!'s CSS Compressor. Removes comments and \
+                    compresses whitespace from CSS files.
+homepage            http://github.com/sprymix/csscompressor
+
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+python.versions     27 37
+
+if {${name} ne ${subport}} {
+    depends_lib-append \
+                    port:py${python.version}-setuptools
```

Comment:
sorry to review so late.... I don't see ```setuptools``` being needed at runtime (i.e., it doesn't install a ```console_script``` or something similar). Are you sure it isn't only needed as a ```build``` dependency? If that's the case, please change the dependency type and it should be ready to merge!

Reply:
I did not notice that: you're right, it should be a build dependency only. Thank you for the review! Fix incoming shortly.